assets/src/components/amp-support/plugins.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { ListItems } from '../list-items';
+
+/**
+ * To render plugins information on site support page.
+ *
+ * @param {Object} props      Component props.
+ * @param {Object} props.data Plugins data.
+ * @return {JSX.Element|null} HTML markup for plugins data.
+ */
+export function Plugins( { data: plugins } ) {
+	if ( 'object' !== typeof plugins ) {
+		return null;
+	}
+
+	plugins = Object.values( plugins );
+
+	return (
+		<details open={ false }>
+			<summary>
+				{ __( 'Plugins', 'amp' ) }
+				{ ` (${ plugins.length || 0 })` }
+			</summary>
+			<div className="detail-body">
+				<ListItems
+					className="list-items--list-style-disc"
+					items={ plugins.map( ( item ) => {
+						return { value: `${ item.name } ${ item.version ? '(' + item.version + ')' : '' }` };
+					} ) }
+				/>
+			</div>
+		</details>
+	);
+}
+
+Plugins.propTypes = {
+	data: PropTypes.array.isRequired,
```

Comment:
```suggestion
	data: PropTypes.object.isRequired,
```

Reply:
We need to keep it an array and not an object. Since we are passing array.

Reply:
Oh Yes, Previously, I did that to assure there is no object (even if then it will be converted into the array) and only an array. But now it will no longer be needed.

I have removed that part.